clients/cobol-lsp-vscode-extension/src/services/ProfileService.ts

```diff
@@ -12,26 +12,28 @@
  *   Broadcom, Inc. - initial API and implementation
  */
 
-import { IProfile } from "@zowe/imperative";
+import {IProfile} from "@zowe/imperative";
 import * as path from "path";
 import * as vscode from "vscode";
-import { Disposable } from "vscode-languageclient";
-import { SETTINGS_SECTION } from "../constants";
-import { ProfilesMap, ZoweApi } from "./ZoweApi";
+import {Disposable} from "vscode-languageclient";
+import {SETTINGS_SECTION} from "../constants";
+import {ProfilesMap, ZoweApi} from "./ZoweApi";
 
 const DEFAULT_STATUS_TEXT = "CPY profile: undefined";
 
 export class ProfileService implements Disposable {
+
+    private static isCobolProgram(fsPath: string) {
+        const ext = path.extname(fsPath).toLocaleLowerCase();
+        return ext === ".cbl" || ext === ".cob" || ext === ".cobol";
```

Comment:
good candidates for reusing, can be extracted as constants

Reply:
you mean the file extensions?

Reply:
Yes